core/spec/models/spree/stock/splitter/weight_spec.rb

```diff
@@ -6,12 +6,15 @@ module Splitter
       describe Weight, :type => :model do
         let(:packer) { build(:stock_packer) }
         let(:variant) { build(:base_variant, :weight => 100) }
+        let(:heavy_variant) { build(:base_variant, :weight => 49) }
```

Comment:
Oops, I flipped these and forgot the names, I'll get that taken care of. I'll squash this change into one commit and the hash syntax change into another so that the changesets are clear.